src/pages/workspace/WorkspacePageWithSections.js

```diff
@@ -82,70 +82,71 @@ const defaultProps = {
+                subtitleKey={_.isEmpty(props.policy) ? undefined : 'workspace.common.notAuthorized'}
             >
-                <FullPageNotFoundView
-                    onBackButtonPress={() => Navigation.goBack(ROUTES.SETTINGS_WORKSPACES)}
-                    shouldShow={_.isEmpty(this.props.policy) || !Policy.isPolicyOwner(this.props.policy)}
-                    subtitleKey={_.isEmpty(this.props.policy) ? undefined : 'workspace.common.notAuthorized'}
-                >
-                    <HeaderWithBackButton
-                        title={this.props.headerText}
-                        subtitle={policyName}
-                        shouldShowGetAssistanceButton
-                        guidesCallTaskID={this.props.guidesCallTaskID}
-                        onBackButtonPress={() => Navigation.goBack(this.props.backButtonRoute || ROUTES.getWorkspaceInitialRoute(policyID))}
-                    />
-                    {this.props.shouldUseScrollView ? (
-                        <ScrollViewWithContext
-                            keyboardShouldPersistTaps="handled"
-                            style={[styles.settingsPageBackground, styles.flex1, styles.w100]}
-                        >
-                            <View style={[styles.w100, styles.flex1]}>{this.props.children(hasVBA, policyID, isUsingECard)}</View>
-                        </ScrollViewWithContext>
-                    ) : (
-                        this.props.children(hasVBA, policyID, isUsingECard)
-                    )}
-                    {this.props.footer}
-                </FullPageNotFoundView>
-            </ScreenWrapper>
-        );
-    }
+                <HeaderWithBackButton
+                    title={props.headerText}
+                    subtitle={policyName}
+                    shouldShowGetAssistanceButton
+                    guidesCallTaskID={props.guidesCallTaskID}
+                    onBackButtonPress={() => Navigation.goBack(props.backButtonRoute || ROUTES.getWorkspaceInitialRoute(policyID))}
+                />
+                {props.shouldUseScrollView ? (
+                    <ScrollViewWithContext
+                        keyboardShouldPersistTaps="handled"
+                        style={[styles.settingsPageBackground, styles.flex1, styles.w100]}
+                    >
+                        <View style={[styles.w100, styles.flex1]}>{props.children(hasVBA, policyID, isUsingECard)}</View>
+                    </ScrollViewWithContext>
+                ) : (
+                    props.children(hasVBA, policyID, isUsingECard)
+                )}
+                {props.footer}
+            </FullPageNotFoundView>
+        </ScreenWrapper>
+    );
 }
 
 WorkspacePageWithSections.propTypes = propTypes;
 WorkspacePageWithSections.defaultProps = defaultProps;
+WorkspacePageWithSections.displayName = 'WorkspacePageWithSections';
 
 export default compose(
     withLocalize,
```

Comment:
Hello, @fedirjh 
For using `useOnNetworkReconnect` hook, if we use this hook, then we can't make fetchData function as useCallback because of React Hooks must be called in a React function component or a custom React Hook function.

Reply:
Because `fetchData` is using props.

Reply:
@fedirjh 
If we use the `useOnNetworkReconnect`, we can't handle `shouldSkipVBBACall`.

```
        // Line 90
        if (shouldSkipVBBACall) {
            return;
        }
```
What do you think?

Reply:
@fedirjh 
Can we remove `network` from propsType?
It is not used if we use the `useOnNetworkReconnect`.

Reply:
@fedirjh 
Updated!
Please check again and let me know your thought.
Thanks.